types/braintree/index.d.ts

```diff
@@ -257,7 +257,7 @@ declare namespace braintree {
         streetAddress?: string;
     }
 
-    export interface AddressUpdateRequest {
+    export type AddressUpdateRequest = [string, string, {
```

Comment:
The tuples look cool, but the main thing that bothers me is not knowing what the arguments are. In this case, you know they're strings, but if you looked at this you wouldn't know what is supposed to be passed in. Maybe comments would help with that?

Reply:
Yeah, I need to add comments for this lib. Will add for the new tuples to help be clearer. I wish there was a way to alias the names. I still think the tuples are a better solution since the request object is now one type.

Reply:
Can you do something like:
```
type BrainTreeThing = string;
type BrainTreeOtherThing = string;
...
export type AddressUpdateRequest = [BrainTreeThing, BrainTreeOtherThing ...
```